pkg/cloudprovider/cloudapi/aws/aws_crd_helpers.go

```diff
@@ -105,8 +106,9 @@ func ec2InstanceToInternalVirtualMachineObject(instance *ec2.Instance, vpcs map[
 	}
 
 	labelsMap := map[string]string{
-		labels.CloudAccountName:      account.Name,
-		labels.CloudAccountNamespace: account.Namespace,
+		labels.CloudAccountName:      accountNamespacedName.Name,
+		labels.CloudAccountNamespace: accountNamespacedName.Namespace,
+		labels.CloudSelectorName:     selectorNamespacedName.Name,
```

Comment:
Do we need the Selector namespace to be displayed in the VM object?
I know the namepsace would be same as VM namespace.

Reply:
do you see the need? I don't think so as our documentation clearly states that vms are imported in ces namespace.